core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/MetricsServiceITest.java

```diff
@@ -867,7 +867,7 @@ public void testBucketPercentiles() {
 
         NumericBucketPoint bucket = actual.get(0);
 
-        assertEquals(testSize, bucket.getSamples());
+        assertEquals((Integer) testSize, bucket.getSamples());
```

Comment:
But why didn't it fail before?

2016-04-27 17:09 GMT+02:00 jsanda <notifications@github.com>:

> In
> core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/MetricsServiceITest.java
> <https://github.com/hawkular/hawkular-metrics/pull/494#discussion_r61276170>
> :
>
> > @@ -867,7 +867,7 @@ public void testBucketPercentiles() {
> >
> >          NumericBucketPoint bucket = actual.get(0);
> >
> > -        assertEquals(testSize, bucket.getSamples());
> > +        assertEquals((Integer) testSize, bucket.getSamples());
>
> Without the cast, it is ambiguous and results in a compiler error. I
> suppose alternatively I could have changed testSize to be an Integer.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/hawkular/hawkular-metrics/pull/494/files/76bb1e94240f40350128c184b599a42ebf2c712f#r61276170>
>



-- 
Thomas Segismont
JBoss ON Engineering Team